tests/local/test_library.py

```diff
@@ -343,42 +343,44 @@ def test_find_exact_any(self):
         result = self.find_exact(any=['local:track:path1'])
         self.assertEqual(list(result[0].tracks), self.tracks[:1])
 
+    # TODO: This is really just a test of the query validation code now,
+    # as this code path never even makes it to the local backend.
```

Comment:
Does that mean that there is some validation code in local that could be removed now?